test/unit/Perspective.spec.ts

```diff
@@ -0,0 +1,78 @@
+import Flicking from "@egjs/flicking";
+
+import Perspective from "../../src/Perspective";
+import { sandbox, cleanup, waitEvent } from "../unit/utils";
+
+describe("Perspective", () => {
+  let flicking: Flicking;
+
+  beforeEach(() => {
+    const wrapper = sandbox("flick");
+    wrapper.style.width = "199px";
+    wrapper.className = "flicking-viewport";
```

Comment:
This line is breaking other tests.
As `cleanup` removes elements with the predefined class, changing the class name made it not remove the sandbox elements.
Please change this using `classList.add` or `className +=`

Reply:
Oh, thank you. I will fix!